docs/content/release_notes/csi_rn_changelog_1.7.0.md

```diff
@@ -0,0 +1,6 @@
+# 1.7.0 (June 2021)
+
+IBM® block storage CSI driver 1.7.0 adds new support and enhancements:
+- Now supports the CSI Topology feature
+- New volume replication (remote copy) support for IBM Spectrum Virtualize Family storage systems
+- Additional support for Kubernetes 1.22 and Red Hat® OpenShift® 4.9
```

Comment:
i don't think 4.9 is in plan

Reply:
DONE (was left in by accident)

Reply:
I just use Ctrl+F to search for `4.9` in the `Files changed` tab to find these [accident](https://github.com/IBM/ibm-block-csi-driver/blob/c464c00aea681c9d52f10eae07e0954403d1e8aa/docs/content/release_notes/csi_rn_whatsnew.md#additional-supported-orchestration-platforms-for-deployment)s

Reply:
DONE